js/functions.js

```diff
@@ -0,0 +1,56 @@
+function checkCommentLength(comment, length) {
+  if (comment.length <= length) {
+    return true;
+  }
+  return false;
```

Comment:
Можно упростить, и возвращать сразу результат выражения
```suggestion
  return comment.length <= length;
```